tests/csv/3.10/samplesheet_test.csv

```diff
@@ -0,0 +1,8 @@
+sample,fastq_1,fastq_2,strandedness
```

Comment:
Why can't we have this file in test-datasets too? We are checking out test-datasets as part of the Github Actions already no? Be nice to keep the samplesheet with the raw data.

Reply:
Of course....
Let me fix that...